LibraryAccess/src/shared/createCORSRequest.js

```diff
@@ -0,0 +1,20 @@
+function createCORSRequest(method, url) {
```

Comment:
This function was copy/pasted in a lot of places, so it made sense to just put it in one file and reuse the call. You might also want to look into something like `axios` or `superagent` or even just the regular `fetch`